enterprise/signals.py

```diff
@@ -30,6 +38,15 @@
 
 logger = getLogger(__name__)
 _UNSAVED_FILEFIELD = 'unsaved_filefield'
+INTEGRATED_CHANNELS = [
+    BlackboardEnterpriseCustomerConfiguration,
+    CanvasEnterpriseCustomerConfiguration,
+    CornerstoneEnterpriseCustomerConfiguration,
+    DegreedEnterpriseCustomerConfiguration,
+    Degreed2EnterpriseCustomerConfiguration,
+    MoodleEnterpriseCustomerConfiguration,
+    SAPSuccessFactorsEnterpriseCustomerConfiguration,
+]
```

Comment:
question: I see that this list doesn't include xapi and I'm assuming that's intentional. What is xapi and why would we exclude it?

Reply:
xapi is sort of the forgotten stepchild of the integrated channels 😆 No one's worked on it for quite some time and It doesn't go through the normal transmission sync process as the other channels as it serves more as a way for people to request the integrated channels transmission data rather than receive it from our integration. I tend to think of xapi as a series of api end points rather than the system that pushes content out to integrated customers.